### ISSUE TITLE
Gated on InSilicoPrediction with SEPIO

### ISSUE THREAD

**Issue by larrybabb** [clingen-data-model/clingen-interpretation#129]
@cbizon @bpow The InSIlicoPrediction SEPIO attributes show a "categoricalPrediction" coding attribute and the value set codings are not available or not clear.  There are actually 2 new value sets at the end of  the SEPIO value set page that seem related, but it is not clear which goes with which attribute and it definitely doesn't have the values.  It looks as if it is an unfinished thought.

Leaving our original InSilicoPrediction as is for now.

**Comment by larrybabb** [comment]
@cbizon @bpow  I think MattB was suggesting that you guys create a list of "prediction outcomes" for attribute A099 InSilicoPrediction.prediction, which is currently a string.  I also think he thought we should do one for the "algorithms" or "tools" attribute on InSilicoPred as well as AlleleConservation.

If you think this is not in scope, then please comment and close, otherwise, send me the possible list items and I will add them to the sheet.

**Comment by bpow** [comment]
I think the problem is that the different in silico prediction tools use different categories (represented as strings) to describe their predictions. If we want to enumerate/list these, then we need to make one or more ValueSets (maybe namespaced by the tool) to describe them.
